internal/app/thanos-notif/app.go

```diff
@@ -16,25 +21,192 @@ type Notifier interface {
 }
 
 const (
-	TransferEventABI = "Transfer(address,address,uint256)"
+	ETHDepositInitiatedEventABI      = "ETHDepositInitiated(address,address,uint256,bytes)"
```

Comment:
I think we can use optimism thanos binding: https://github.com/tokamak-network/tokamak-thanos/blob/main/op-bindings/bindings/l1standardbridge.go

In bindings folder, there are many ABIs and helper functions. It will help us a lot. We just import to our project and start using.

Reply:
[PasrseETHBridgeInitiated](https://github.com/tokamak-network/tokamak-thanos/blob/2bbacec3da4393033d98588a006875f80f1fcc5b/op-bindings/bindings/l1standardbridge.go#L1996)

This function looks like useful to parsing Events. it maybe reduce event handler code.